SwifterDemoiOS/AppDelegate.swift

```diff
@@ -24,17 +24,12 @@
 //
 
 import UIKit
-import SwifteriOS
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
-                            
-    var window: UIWindow?
-
-    func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:]) -> Bool {
-        Swifter.handleOpenURL(url)
-        return true
+    func application(_ application: UIApplication, configurationForConnecting connectingSceneSession: UISceneSession, options: UIScene.ConnectionOptions) -> UISceneConfiguration {
```

Comment:
Like the above comment, leave this in here for anyone who isn't targeting iOS 13 and above. UIApplication is still used as a fallback for anyone below iOS 13 (even if you have scenes)

Reply:
Fixed!
https://github.com/mattdonnelly/Swifter/pull/271/commits/7e6f2fc71be0e6caf8d72dec5ccfd3723a11f7f6